src/App.svelte

```diff
@@ -21,15 +22,12 @@
 	});
 
 	onMount(() => {
-		if(walletDataExists()) {
+		if (walletDataExists()) {
```

Comment:
Ups, just added the eslint-plugin to my IDE. Since we're not auto fixing *.svelte this plugin will auto correct to our style on save.